lib/plugins/extension/helper/extension.php

```diff
@@ -204,6 +206,31 @@ public function isTemplate()
         return $this->is_template;
     }
 
+    /**
+     * Set source (remote/local/null) that getters should be forced on
+     *
+     * @var string $source
+     * @return void
+     */
+    public function setForceSource($source)
+    {
+        $this->forceSource = $source;
+    }
+
+    /**
+     * If the source should be ignored for getters
+     *
+     * @var string $source
+     * @return bool If this extension is a template
```

Comment:
This comments seems wrong. Copy'n'paste error?

Reply:
Thanks for catching that.